examples/writeAll-documents.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2021 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+let exutil = require('./example-util.js');
+const marklogic = exutil.require('marklogic');
+const dbWriter = marklogic.createDatabaseClient(exutil.restWriterConnection);
+let uris = [];
```

Comment:
It looks like the uris array isn't used after being populated. 

Could the uris array be removed from the example to simplify and focus attention on the readable?